jazz_assets/components/crud/get.js

```diff
@@ -36,7 +36,7 @@ module.exports = (assets_id, asset_table, onComplete) => {
         if (err) {
             onComplete(err);
         } else {
-            var responseData = utils.formatResponse(data.Item) || (data.Items).map(item => utils.formatResponse(item));
+            var responseData = (data.Item) ? utils.formatResponse(data.Item) : (data.Items).map(item => utils.formatResponse(item));
```

Comment:
you data.Item and data.Items? what is the difference here. its confusing with the naming here.

Reply:
response of docClient.get and docClient.scan have different keyname "Item" and "Items" respectively.